### DISCUSSION TITLE
Custom icons in browser manager sidebar

### DISCUSSION THREAD

**Discussion by DasDunkel** [Floorp-Projects/Floorp#854] [Feature Request]
It would be nice if custom icons could be configured for the browser manager sidebar, I have added extensions to it, such as bitwarden, by setting the website `moz-extension://<UUID>/popup/index.html` but this shows a generic extension icon for all of them, making it challenging to know which is which

**Comment by breadbeforerice** [comment]
I tried bitwarden by adding it from the + button and was able to distinguish it by icon.

**Reply by DasDunkel** [reply]
Extension support in the sidebar was recently added to Floorp, this is technically no longer relevant as you don't need to manually add the URL yourself

**Reply by KaKi87** [reply]
Hi,

How were you able to add any extensions at all ?

Personally, I tried adding [LessPass](https://addons.mozilla.org/fr/firefox/addon/lesspass/), and I'm getting this :

<img width="800" height="1136" alt="Screenshot_20250910_005940" src="https://github.com/user-attachments/assets/870432ad-b6d5-41d9-a95e-8cb432a42d8f" />

Thanks

**Comment by johnsonjamees** [comment]
I'd like to second this feature request. I use Raindrop.io for bookmark management, and the browser manager sidebar uses a dark-colored icon for this extension. Since I also use Floorp in dark mode, the icon's visibility is an issue. Allowing users to specify their own custom icons would help resolve this problem.

**Reply by breadbeforerice** [reply]
You can specify icons using userChrome.css.
